indico/modules/attachments/templates/mako_compat/attachments_tree.html

```diff
@@ -0,0 +1,16 @@
+{% from 'attachments/_attachments.html' import render_attachments %}
+
+<div class="attachments-box {%- if can_edit %} highlighted-area{% endif %}">
+    <div class="attachments-box-header">
+        <h2 class="icon-attachment"> {% trans %}Materials{% endtrans %}</h2>
```

Comment:
is the space needed? if yes let's add a jinja comment or someone will probably remove it at some point, thinking it's useless